packages/api/src/resolvers/mutations/accounts/stopImpersonation.ts

```diff
@@ -0,0 +1,31 @@
+import { UserNotFoundError } from '@unchainedshop/api/errors.js';
+import { log } from '@unchainedshop/logger';
+import { Context, Root } from '@unchainedshop/types/api.js';
+
+export default async function stopImpersonation(root: Root, _, context: Context) {
+  const { userId, modules, loginToken } = context;
+
+  log(`mutation stopImpersonation for ${userId}`);
+
+  const impersonatedUser = await modules.users.findUserById(userId);
+
+  if (!impersonatedUser) throw new UserNotFoundError({ userId });
+  const impersonator = impersonatedUser?.services?.resume?.loginTokens.find(
```

Comment:
Remark: This naming is missleading, one could think the object holds the user object of the impersonator but it actually holds the loginToken of the impersonated user.